hw/bsp/stm32f3discovery/src/hal_bsp.c

```diff
@@ -50,6 +50,19 @@
 #include "mcu/mcu.h"
 #include "os/os_cputime.h"
 
+#if MYNEWT_VAL(LSM303DLHC_ONB)
+#include "lsm303dlhc/lsm303dlhc.h"
+static struct lsm303dlhc lsm303dlhc;
+#endif
+
+#if MYNEWT_VAL(LSM303DLHC_ONB)
+static struct sensor_itf i2c_0_itf_lis = {
```

Comment:
I wander where this _lis came from?

Reply:
I copied it originally from another BSP that uses some lis something. I removed the suffix cause it is not relevant in this context.